docs/rules/arrow-body-style.md

```diff
@@ -20,15 +20,17 @@ The second one is an object for more fine-grained configuration when the first o
 "arrow-body-style": ["error", "always"]
 ```
 
-When the rule is set to `"always"` the following patterns are considered problems:
+### always
+
+Examples of **incorrect** code for this rule with the `"always"` option:
 
 ```js
 /*eslint arrow-body-style: ["error", "always"]*/
 /*eslint-env es6*/
 let foo = () => 0;
```

Comment:
No worries, you can concentrate on headings and sentences. How about indicate which you do now with thumbs up emoji.

Brain is fried now, maybe we can meet on gitter over the week end to sort out the branches. Doing procedural stuff offline might help keep from cluttering the issue.